eZ Platform/APIs/Languages/README.md

```diff
@@ -0,0 +1,70 @@
+# v2: Updated API Language behaviour
+
+## Objective
+
+Simplify usage of the product for developers, avoid common mistakes when dealing with multi language installs,
+and make sure to cover needs for future UI needs.
+
+### Use Cases
+The identified new behaviour:
+- As a Developer, I want to get API entities in prioritised language of the given SiteAccess,
+  so I don't need to specify language in API Service calls
+- As a Developer, I want to get API entities in one Translation at a time,
+  so I don't need to specify language or use helpers when using the API entities
+
+##### Related use cases
+- As an Editor, in UI I want to be able to select languages of API entities to view and edit translation independently
+  of the current SiteAccess, _so I don't need to deal with several variants of backend to do that_
```

Comment:
So the first use case is just partly implemented in 1.x, is that what you mean? For content objects it works (though we might have bugs), but clearly not for users or content type Am I right

Can I suggest a rephrase:

> As an Editor, in eZ Platform UI I want to be able to select the default language of entities coming from the repository API (e.g. content, content types, user, user group...) independently of the current SiteAccess so I don't need to access different instances of Platform UI to view those entities in different languages.
> As an Editor, in Platform UI I want to be able to select another language for managing entities through the repository API (e.g. content, content types, user, user group...) so that I can create update or read those entities in different language without having to access different instances of Platform UI.
> As an Editor, I want to be able to select the language of eZ Platform UI interface (e.g. menus, labels, buttons... which are not coming from the repository as opposode to entities mentioned above) independently of the current SiteAccess  so that I can use the UI in different languages without having to access different instances of Platform UI.

_note: the 2nd use case is a no brainer for content, but I am unsure if it needs to be implemented for all kind of entities._